src/BuildPrediction/Predictors/CopyTask/CopyTaskPredictor.cs

```diff
@@ -36,22 +36,20 @@ public class CopyTaskPredictor : IProjectPredictor
             var activeTargets = new Dictionary<string, ProjectTargetInstance>(StringComparer.OrdinalIgnoreCase);
 
             // Start with the default Build target and all of its parent targets, the closure of its dependencies.
-            project.AddToActiveTargets(MsBuildHelpers.BuildTargetAsCollection, activeTargets);
+            projectInstance.AddToActiveTargets("Build", activeTargets);
```

Comment:
Is this available in a const somewhere in this repo or in MSBuild?

Reply:
Nope, in fact I'll change this to check `DefaultTargets` and `InitialTargets`. MSBuild itself doesn't actually use a hardcoded "Build", it just happens to be the first defined target, or projects define it as the `DefaultTargets`.

VS on the other hand I believe does actually hard-code target names.

Reply:
From DefaultTargets docs:
> DefaultTargets specified in the project, or the logically first target if no
> DefaultTargets is specified in the project. The build builds these if no targets
> are explicitly specified to build.